mindsdb_native/libs/phases/model_interface/lightwood_backend.py

```diff
@@ -564,6 +572,9 @@ def predict(self, mode='predict', ignore_columns=None, all_mixers=False):
                     continue
 
                 formated_predictions[k] = predictions[k]['predictions']
+                if 'class_distribution' in predictions[k]:
```

Comment:
Better to check `if self.transaction.lmd['output_class_distribution']` maybe, to avoid certain silent failures.